src/extension/provider/statefulAuth.ts

```diff
@@ -248,6 +237,90 @@ export class StatefulAuthProvider implements AuthenticationProvider, Disposable
     this.#disposables.forEach((d) => d.dispose())
   }
 
+  public async bootstrapFromToken() {
+    const authTokenUri = Uri.joinPath(this.context.extensionUri, 'secrets', 'authToken')
```

Comment:
As said in the first review, the location of the token should be configurable and default to the location hardcoded here.